From 6934173c70fe5de1841efb50e15228e2b251afeb Mon Sep 17 00:00:00 2001
From: Christian Kohn <christian.kohn@xilinx.com>
Date: Wed, 11 Mar 2015 11:13:28 -0700
Subject: [PATCH 627/827] drm: xilinx: plane: Do not set background color in DRM_MODE_DPMS_ON

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Don't reset the background color but use default value that the
user selects when instantiating the IP core. In the long run
this should be a separate plane property.

Signed-off-by: Christian Kohn <christian.kohn@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit b2454e8eff0a6547f9a91d1eee68698f31e5bb97)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/xilinx/xilinx_drm_plane.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/xilinx/xilinx_drm_plane.c b/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
index f0c39b5..3b90d1c 100644
--- a/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
+++ b/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
@@ -609,7 +609,6 @@ void xilinx_drm_plane_manager_dpms(struct xilinx_drm_plane_manager *manager,
 
 		if (manager->osd) {
 			xilinx_osd_disable_rue(manager->osd);
-			xilinx_osd_set_color(manager->osd, 0x0, 0x0, 0x0);
 			xilinx_osd_enable(manager->osd);
 			xilinx_osd_enable_rue(manager->osd);
 		}
-- 
2.9.3

